packages/accounts-passwordless/passwordless_server.js

```
@@ -114,7 +106,13 @@ function generateSequence() {
 }
 
 Meteor.methods({
-  requestLoginTokenForUser: async ({ selector, userData, options = {} }) => {
+  requestLoginTokenForUser: async (args) => {
```

ESLint

[error] Replace `(args)` with `args`

(prettier/prettier)